.coderabbit.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: "en-US"
+early_access: false
+reviews:
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_walkthrough: true
+  auto_review:
+    enabled: true
+    ignore_title_keywords:
+      - "WIP"
+      - "DO NOT MERGE"
+    drafts: false
+    base_branches:
```

Comment:
What does this mean? Only review PR to those branches?

Reply:
Yes, only review the master and main branches. For other branches such as v1.7.x and v1.6.x, their PRs are backported, so they don't need to review the PRs again. WDYT?